src/main/java/application/rest/v1/actions/ResolutionContext.java

```diff
@@ -358,13 +359,16 @@ private void initializeResourceMap() {
     }
     
     public String getConfigMapDataField(String mapName, String mapField) {
-        if (kappnavNSMapCache.containsKey(mapName)) {
-            // Return value from the local cache.
-            final V1ConfigMap map = kappnavNSMapCache.get(mapName);
-            if (map != null) {
-                return map.getData().get(mapField);
+        // if CONFIGMAP_CACHING set to disable then do not look from cache already read it
```

Comment:
Please use this comment:  

// read from cache only if enabled - use case for disabling is for pattern testing